README.md

```diff
@@ -19,7 +19,7 @@ This is a simple Node.js library for interacting with the [Vimeo API](https://de
 
 ## Get started with the Vimeo API
 
-There is a lot of information about the Vimeo API at https://developer.vimeo.com/api/start. Most of your questions will be answered there!
+There is a lot of information about the Vimeo API at https://developer.vimeo.com/api/start. Most of your questions are answered there!
 
 ## Direct Help
```

Comment:
Can we remove the Google Group link? We no longer monitor or support it.

Reply:
Sure thing. I'll pull it from all three libraries.